tests/tests/constants/contractSources.ts

```diff
@@ -0,0 +1,65 @@
+export const contractSources: { [key: string]: string } = {
+  // Solidity: contract test {function multiply(uint a) public pure returns(uint d) {return a * 7;}}
+  TEST_CONTRACT: `
```

Comment:
Isn't possible to include it in the json, so we have both in 1 place ?

Reply:
To include what in the json? The source code?

Reply:
yes the contract source code